controllers/controllers.ts

```diff
@@ -164,6 +164,55 @@ export async function getOrgAppsPage(
   };
 }
 
+export async function getOrgUsagePage(
+  orgGuid: string
+): Promise<ControllerResult> {
+  const [orgQuotasRes, orgUsageRes, svcInstancesRes] = await Promise.all([
+    CF.getOrgQuotas({ organizationGuids: [orgGuid] }),
+    CF.getOrgUsageSummary(orgGuid),
+    CF.getServiceInstances({ organizationGuids: [orgGuid] }),
+  ]);
+  [orgQuotasRes, orgUsageRes, svcInstancesRes].map((res) => {
+    if (!res.ok) {
+      logDevError(
+        `api error on cf org usage page with http code ${res.status} for url: ${res.url}`
+      );
+      throw new Error('something went wrong with the request');
+    }
+  });
+
+  const quotas = await orgQuotasRes.json();
+  const svcInstances = await svcInstancesRes.json();
+  const svcPlanGuids = svcInstances.resources.map(function (instance: any) {
```

Comment:
`instance: any` this should probably have a TS interface, since we're relying on it having specific params.

Reply:
I'll go ahead and add a couple of the responses in, since we know what the API response should look like for all of them!